gatsby/src/styles/global.scss

```diff
@@ -1,32 +1,25 @@
-@use '@nice-digital/nds-core/scss/core';
+@use '@nice-digital/nds-core/scss/spacing';
+@use '@nice-digital/nds-core/scss/utils';
 
 $nds-include-spacing-modifiers: false;
```

Comment:
I think these variables have changed names, and can be configured with the import below so they're like:

```
@use '@nice-digital/design-system/scss/base' with (
  $include-spacing-modifiers: false,
  $output-text-classes: false,
  $output-visibility-classes: false
);
```